Jenkinsfile

```diff
@@ -4,6 +4,13 @@ String tfComponent   = "OneOneOne"  // this defines the application - nhais, mhs
 
 Map<String,String> tfOutputs = [:] //map for collecting values from tfOutput
 
+Map<String,String> nginxCerts = [
+  "ca.cer": "arn:aws:secretsmanager:eu-west-2:067756640211:secret:nginx-111-ca-cer-iE9CDY",
```

Comment:
Please add these in as Jenkins env vars to remove the ARNs from here

Reply:
Removed, the build process does not require the certs any more - they are supplied during runtime.